api/provider/aws/service_delete_test.go

```diff
@@ -0,0 +1,38 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/quintilesims/layer0/api/tag"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func TestService_deleteService(t *testing.T) {
+
+	tagStore := tag.NewMemoryStore()
+	serviceProvider := NewServiceProvider(nil, tagStore, nil)
+	_ = serviceProvider
+
+	tags := models.Tags{
+		{
+			EntityID:   "sID",
+			EntityType: "service",
+			Key:        "name",
+			Value:      "sName",
+		},
+		{
+			EntityID:   "eID",
+			EntityType: "environment",
+			Key:        "name",
+			Value:      "eName",
+		},
+	}
+
+	for _, tag := range tags {
+		if err := tagStore.Insert(tag); err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	// TODO: Add delete service test.
```

Comment:
I think here, you can make a call to the common helper function `deleteServiceTags()` and pass in `tagStore, "service", serviceID` where serviceID would be something like `serviceID := "sID"`, then make assertions after the call that the tags database has removed the entity. Once service Create() has been implemented, if Create() inserts other `models.Tags` into the tags db, you can add them to `tags` later and test to make sure that all associated `models.Tags` are removed.

Reply:
Sorry, I mean the common function `deleteEntityTags`, not `deleteServiceTags()`

Reply:
@jparsons04 thanks, I'll take a look at this.